### ISSUE TITLE
BUG: No files are created.

### ISSUE THREAD

**Issue by ZeroDot1** [funilrys/PyFunceble#274]
## Description
No lists are created.

## Reproduction

1. Install the latest dev version
2. Run pyfunceble -w 32 --plain -a --syntax -f FILE.TXT
3. Open the folders and you can' t find any files.

## Expected behavior

That the files are created and are accessible.

## Screenshots

![Bildschirmfoto vom 2021-11-19 12-04-23](https://user-images.githubusercontent.com/28985171/142612431-704bc032-c2de-45ae-8bc2-3a13c1945773.png)
![Bildschirmfoto vom 2021-11-19 12-08-20](https://user-images.githubusercontent.com/28985171/142612899-d93f1223-e29b-427e-bd59-741d7665e2cb.png)

## Versions

**OS:** Arch Linux (5.17-r4213121-1-ARCH) -->

**Python Version:** 3.9.7-2

**PyFunceble Version:** 4.0.2b5.dev (Blue Duckling: Chestnut)

**Comment by ZeroDot1** [comment]
```py
[user@work ~]$ sudo pip install pyOpenSSL
[sudo] Passwort für user: 
Requirement already satisfied: pyOpenSSL in /usr/lib/python3.9/site-packages (20.0.1)
Requirement already satisfied: cryptography>=3.2 in /usr/lib/python3.9/site-packages (from pyOpenSSL) (35.0.0)
Requirement already satisfied: six>=1.5.2 in /usr/lib/python3.9/site-packages (from pyOpenSSL) (1.15.0)
Requirement already satisfied: cffi>=1.12 in /usr/lib/python3.9/site-packages (from cryptography>=3.2->pyOpenSSL) (1.14.6)
Requirement already satisfied: pycparser in /usr/lib/python3.9/site-packages (from cffi>=1.12->cryptography>=3.2->pyOpenSSL) (2.20)
[user@work ~]$ 

```

**Comment by ZeroDot1** [comment]
```py
/home/user/.local/lib/python3.9/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.25.11) or chardet (4.0.0) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
/usr/lib/python3.9/site-packages/cryptography/hazmat/backends/openssl/x509.py:15: CryptographyDeprecationWarning: This version of cryptography contains a temporary pyOpenSSL fallback path. Upgrade pyOpenSSL now.
  warnings.warn(
The 'collection.preferred_status_origin' key is missing from your configuration file.
Are we authorized to merge it from upstream ? [y/n] y

    ██████╗ ██╗   ██╗███████╗██╗   ██╗███╗   ██╗ ██████╗███████╗██████╗ ██╗     ███████╗
    ██╔══██╗╚██╗ ██╔╝██╔════╝██║   ██║████╗  ██║██╔════╝██╔════╝██╔══██╗██║     ██╔════╝
    ██████╔╝ ╚████╔╝ █████╗  ██║   ██║██╔██╗ ██║██║     █████╗  ██████╔╝██║     █████╗
    ██╔═══╝   ╚██╔╝  ██╔══╝  ██║   ██║██║╚██╗██║██║     ██╔══╝  ██╔══██╗██║     ██╔══╝
    ██║        ██║   ██║     ╚██████╔╝██║ ╚████║╚██████╗███████╗██████╔╝███████╗███████╗
    ╚═╝        ╚═╝   ╚═╝      ╚═════╝ ╚═╝  ╚═══╝ ╚═════╝╚══════╝╚═════╝ ╚══════╝╚══════╝
    
Process pyfunceble_tester_worker_20:
Traceback (most recent call last):
  File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.9/site-packages/PyFunceble/cli/processes/workers/base.py", line 273, in run
    PyFunceble.facility.ConfigLoader.set_custom_config(self.configuration)
  File "/usr/lib/python3.9/site-packages/PyFunceble/config/loader.py", line 209, in set_custom_config
    self.custom_config = value
  File "/usr/lib/python3.9/site-packages/PyFunceble/config/loader.py", line 135, in wrapper
    self.start()
  File "/usr/lib/python3.9/site-packages/PyFunceble/config/loader.py", line 385, in start
    config = self.get_config_file_content()
  File "/usr/lib/python3.9/site-packages/PyFunceble/config/loader.py", line 322, in get_config_file_content
    config = self.dict_helper.from_yaml_file(self.path_to_config)
  File "/usr/lib/python3.9/site-packages/PyFunceble/helpers/dict.py", line 291, in from_yaml_file
    data = yaml_load(file_stream)
  File "/usr/lib/python3.9/site-packages/yaml/__init__.py", line 162, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib/python3.9/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/usr/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/usr/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/usr/lib/python3.9/site-packages/yaml/scanner.py", line 115, in check_token
    while self.need_more_tokens():
  File "/usr/lib/python3.9/site-packages/yaml/scanner.py", line 152, in need_more_tokens
    self.stale_possible_simple_keys()
  File "/usr/lib/python3.9/site-packages/yaml/scanner.py", line 291, in stale_possible_simple_keys
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "/home/user/.config/PyFunceble/.PyFunceble.yaml", line 152, column 1
could not find expected ':'
  in "/home/user/.config/PyFunceble/.PyFunceble.yaml", line 153, column 1

```

**Comment by spirillen** [comment]
Nice set of reports It strikes me if you have set the `--no-files` in your `.PyFunceble.overwrite.yaml`

typical location `~/.config/PyFunceble/.PyFunceble.overwrite.yaml`

Did you try to compile python from source as I mentioned in another issue some days ago, I had the same issues as you have here, but they disappeared (almost) once I install a self-compiled python 3.10. Not saying it is **_the_** solution but might be **_one_** solution

**Comment by ZeroDot1** [comment]
Hm @spirillen i have not set the `--no-files` flag.

Unfortunately I don't have the time to compile python myself at the moment, I lost my home a few months ago through no fault of my own, and currently have to work between door and angel.
I'm very happy that I can at least provide some updates for the CoinBlockerLists and especially my security IP feeds.

**Comment by funilrys** [comment]
I'm wondering if you could post the state of you `.PyFunceble.yaml` file-s after merging !

**Comment by ZeroDot1** [comment]
> I'm wondering if you could post the state of you `.PyFunceble.yaml` file-s after merging !

[PyFunceble_Config_ZD.zip](https://github.com/funilrys/PyFunceble/files/7571592/PyFunceble_Config_ZD.zip)

Hi @funilrys see the Zip-File

**Comment by spirillen** [comment]
>  currently have to work between door and angel.

Very sad to hear. If it wasn't for the 500 km you could crash on my couch. Hoping you'll soon get a more permanent place to stay, i know what it is to be broke..
What happens if you moves `.PyFunceble.overwrite.yaml` to `.PyFunceble.overwrite.yaml.bak` and starts out on a empty one? as I don't spot anything that should course this error

**Comment by funilrys** [comment]
Uhh, the Scan Error typically means that the configuration files are not YAML compliant ... 

But yours is defenitely YAML compliant ...
I copied your configuration files in my environment and it's working .... 

What's the content of `/home/user/.config/PyFunceble/.PyFunceble.yaml` ?

**Comment by funilrys** [comment]
> currently have to work between door and angel.

Very sad to read 😞. I hope (too) that everything will get sorted soon. I can't imagine how bad it is but my hope and prayers will be with/for you.

**Comment by spirillen** [comment]
That one is in the zip (https://github.com/funilrys/PyFunceble/issues/274#issuecomment-974225313) as well

![image](https://user-images.githubusercontent.com/44526987/142725978-680c031a-41ab-4a97-9110-0dbe3e927f59.png)
/home/ZeroDot1/.config/PyFunceble/
<details><summary>/home/ZeroDot1/.config/PyFunceble/.PyFunceble.yaml</summary>

```yaml
cli_decoding:
    adblock: false
    adblock_aggressive: false
    rpz: false
    wildcard: false
cli_testing:
    autocontinue: false
    ci:
        active: false
        branch: master
        command: null
        commit_message: PyFunceble - AutoSave
        distribution_branch: master
        end_command: null
        end_commit_message: PyFunceble - Results
        max_exec_minutes: 15
    cidr_expand: false
    complements: false
    cooldown_time: 0.0
    days_between:
        db_clean: 28
        db_retest: 1
    db_type: csv
    display_mode:
        all: false
        colour: true
        dots: false
        execution_time: false
        less: true
        percentage: true
        quiet: false
        simple: false
        status: all
    file_filter: null
    file_generation:
        analytic: true
        hosts: false
        merge_output_dirs: false
        no_file: false
        plain: true
        unified_results: false
    hosts_ip: 0.0.0.0
    inactive_db: true
    local_network: false
    max_workers: null
    mining: false
    preload_file: false
    sorting_mode:
        hierarchical: false
        standard: true
    testing_mode:
        availability: true
        reputation: false
        syntax: false
    whois_db: true
collection:
    preferred_status_origin: recommended
    push: false
    url_base: https://collection.dead-hosts.funilrys.com
debug:
    active: false
    level: info
dns:
    delay: 0.0
    follow_server_order: true
    protocol: UDP
    server: null
    trust_server: false
http_codes:
    list:
        potentially_down:
        - 400
        - 402
        - 404
        - 409
        - 410
        - 412
        - 414
        - 415
        - 416
        - 451
        potentially_up:
        - 0
        - 300
        - 301
        - 302
        - 303
        - 304
        - 305
        - 307
        - 308
        - 403
        - 405
        - 406
        - 407
        - 408
        - 411
        - 413
        - 417
        - 418
        - 421
        - 422
        - 423
        - 424
        - 426
        - 428
        - 431
        - 500
        - 501
        - 502
        - 503
        - 504
        - 505
        - 506
        - 507
        - 508
        - 510
        - 511
        up:
        - 100
        - 101
        - 102
        - 200
        - 201
        - 202
        - 203
        - 204
        - 205
        - 206
        - 207
        - 208
        - 226
        - 429
    self_managed: false
links:
    api_date_format: https://pyfunceble.funilrys.com/api/date-format
    api_no_referrer: https://pyfunceble.funilrys.com/api/no-referrer
lookup:
    collection: false
    dns: true
    http_status_code: true
    netinfo: true
    reputation: false
    special: true
    timeout: 5
    whois: true
share_logs: false
user_agent:
    browser: chrome
    custom: null
    platform: linux
verify_ssl_certificate: false

```

<details>

<details><summary>/home/ZeroDot1/.config/PyFunceble/.PyFunceble.overwrite.yaml</summary>

```yaml
debug:
  # Provides everything related to the debug mode.

  # Activates the debug/logging.
  active: False
  # Sets the logging level.
  level: info

cli_decoding:
  # Provides everything related to the decoding from the CLI.

  # Activates the aggressive decoding.
  adblock_aggressive: False

  # Activates the decoding using the adblock decoder.
  adblock: False

  # Activates the decoding using the RPZ decoder.
  rpz: False

  # Activated the decoding using the Wildcard decoder.
  wildcard: False

cli_testing:
  # Provides everything directly related to the CLI testing.

  # Sets the Hosts IP to use while generation the hosts file(s)
  hosts_ip: "0.0.0.0"

  # Sets the number of maximal workers to use.
  # If set to null, the system use: CPU * Cores - 2
  max_workers: null

  # Activates the automatic continuation after a break or shortage.
  autocontinue: False

  # Activates the storage and creation of a database of all INACTIVA and INVALID
  # subjects for continuous testing.
  inactive_db: False

  # Activates the storage of some WHOIS information in order to avoid having
  # to spam the WHOIS servers.
  whois_db: True

  # Activates the generation of complements.
  complements: False

  # Activates the expansion of CIDR to single addresses.
  cidr_expand: False

  # Sets the cooldown time to apply between each tests.
  cooldown_time: 0.0

  # Sets the Database Connector type to use.
  # Available: csv | mariadb | mysql
  db_type: csv

  # Sets the filter to apply while reading the given input.
  # For example, if you give `\.info`, we will only test the subjects who match
  # `\.info`.
  file_filter: null

  # Activates the mining of data.
  mining: False

  # Acknowledges that we may test for local network component.
  # NOTE: Activating this, will remove the syntax checker completely.
  local_network: False

  # Activates or disable the preloading of the given input file(s).
  # When this is activated, we take the time to pre load and decode the content
  # of your file so that we can optimize some of our process regarding the
  # auto continue.
  #
  # Note: This has no effect if the auto-continue is deactivated.
  preload_file: True

  ci:
    # Provides everything related to the Continuous integration.

    # Activates the continuous integration mode.
    # WARNING: Do not activate without asking or knowing what you are doing.
    active: False

    # Sets the commit message to apply everytime except for the last one.
    commit_message: "PyFunceble - AutoSave"

    # Sets the commit message to apply at the very end of the test.
    end_commit_message: "PyFunceble - Results"

    # Sets the number of minutes  to wait before starting to save and stop
    # a session.
    max_exec_minutes: 15

    # Sets the working branch. This is the branch from where we are testing.
    branch: master

    # Sets the distribution branch. This is the branch that is going to get
    # the (final) results.
    distribution_branch: master

    # Sets the command to execute everytime before each commits except the last
    # one.
    command: null

    # Sets the command to execute before the last commit.
    end_command: null

  display_mode:
    # Provides everything related to the display mode.

    # Activates the printing of dots.
    dots: False

    # Activates the printing of the execution time.
    execution_time: True

    # Activates the output of the percentage information.
    percentage: True

    # Activates the quiet mode.
    quiet: False

    # Activates the output of very few information.
    less: True

    # Activates the output of all information.
    all: False

    # Activates the output of the status and it's status (only)
    simple: False

    # Activates the coloration of stdout.
    colour: False

    # Sets the status do display to STDOUT.
    # NOTE: This does not have any effect with the generated files.
    # Available values: ALL | ACTIVE | INACTIVE | INVALID | SANE | MALICIOUS
    status: all

  testing_mode:
    # Provides all available testing modes.
    # WARNING: Only one can be used at a time.

    # Activates the availability test.
    availability: True

    # Activates the syntax test.
    syntax: False

    # Activates the reputation test.
    reputation: False

  days_between:
    # Provides everything which is x days periodic.

    # NOT IMPLEMENTED (Anticipation for future usage).
    db_clean: 28

    # Provides the number of days to wait before retesting subject which were
    # stored into the inactive database.
    db_retest: 1

  sorting_mode:
    # Provides all our sorting mode.
    # WARNING: The sorting mode is only applied to the files. NOT STDOUT.

    # Activates the hierarchical sorting.
    hierarchical: False

    # Activates the standard sorting.
    standard: True

  file_generation:
    # Provides everything related to the generation of file.

    # Deactivates the generation of any status file.
    no_file: False

    # Activates the generation of the hosts file(s).
    hosts: False

    # Activates the generation of the plain (or raw) file(s).
    plain: True

    # Activates the generation of the analytic file(s).
    analytic: True

    # Activate the generation of an unified file for the copy of what is
    # outputted to STDOUT.
    unified_results: False

    # Activates or disables the merging of the results of all inputted files into
    # one single output subdirectory.
    merge_output_dirs: False

lookup:
  # Provides everything related to the lookup.

  # Activates the usage of the DNS lookup.
  dns: True

  # Activates the usage of the HTTP status code lookup.
  http_status_code: True

  # Activates the usage of the network information socket.
  netinfo: True

  # Activates the usage of our special rules.
  special: True

  # Activates the usage of the WHOIS record.
  whois: False

  # Activates the usage of the reputation data reputation.
  reputation: False

  # Sets the timeout to apply to each of our lookup tools who support a timeout
  # option.
  timeout: 5

dns:
  # Provides everything related to the DNS lookup.

  # Activates the follow-up of the given order.
  follow_server_order: true

  # Activates the trust mode. Meaning that if one of the DNS give us a negative
  # response (without error), we take it as it is.
  # When this mode is deactivated (false), when one of the DNS give us a negative
  # response (without error), we still ask all other servers.
  trust_server: false

  # Sets the list of server to communicate with.
  # It must be a list.
  # As example:
  #   - first.dns
  #   - second.dns
  #
  server: 192.168.3.1

  # Sets the protocol to use.
  # Available: UDP | TCP | HTTPS | TLS
  protocol: TLS

# Not Implemented yet. Reserved for future usage and implementation.
share_logs: False

user_agent:
  # Provides everything related to the user agent choice.
  # Note:
  #   Please report to https://git.io/JLWe5 for more insight about the
  #   browser and platform available..

  browser: chrome
  platform: linux

  # Sets the user agent to use.
  # WARNING: If given, this will be used systematically.
  custom: null

# Activates the verification of the certificate.
verify_ssl_certificate: False

http_codes:
  # Provides everything related to the HTTP code lookup interpretation.

  # Informs PyFunceble that the end-user (not PyFunceble) is managing this list.
  # This means that you won't get any update in this section.
  self_managed: False

  list:
    up:
      - 100
      - 101
      - 102
      - 200
      - 201
      - 202
      - 203
      - 204
      - 205
      - 206
      - 207
      - 208
      - 226
      - 429
    potentially_down:
      - 400
      - 402
      - 404
      - 409
      - 410
      - 412
      - 414
      - 415
      - 416
      - 451
    potentially_up:
      - 000
      - 300
      - 301
      - 302
      - 303
      - 304
      - 305
      - 307
      - 308
      - 403
      - 405
      - 406
      - 407
      - 408
      - 411
      - 413
      - 417
      - 418
      - 421
      - 422
      - 423
      - 424
      - 426
      - 428
      - 431
      - 500
      - 501
      - 502
      - 503
      - 504
      - 505
      - 506
      - 507
      - 508
      - 510
      - 511

links:
  # Provides some of our API links.
  # NOTE: Not implemented yet as the underlaying infrastructure will be
  # rewritten.
  api_date_format: "https://pyfunceble.funilrys.com/api/date-format"
  api_no_referrer: "https://pyfunceble.funilrys.com/api/no-referrer"

```

<details>

**Comment by ZeroDot1** [comment]
OK, I loaded a completely new configuration, but the problem still exists, which means it is not my configuration that is causing the problem.
![Bildschirmfoto vom 2021-11-21 15-25-58](https://user-images.githubusercontent.com/28985171/142765981-660ff708-655b-4134-a78b-3112b7884756.png)
![Bildschirmfoto vom 2021-11-21 15-26-22](https://user-images.githubusercontent.com/28985171/142765988-7a675900-fa57-494b-8b0d-25152d201dd9.png)

**Comment by spirillen** [comment]
could you test on the default test list https://raw.githubusercontent.com/PyFunceble/ci_test/master/test.list and set `--logging-level debug`

and post the log output to @funilrys ?

**Comment by ZeroDot1** [comment]
@spirillen @funilrys I quickly executed it, all data is attached (zip file)
[test.list.zip](https://github.com/funilrys/PyFunceble/files/7576847/test.list.zip)
![Bildschirmfoto vom 2021-11-21 19-39-53](https://user-images.githubusercontent.com/28985171/142774837-d969188a-8f51-4440-aa27-1c22ee1a76b8.png)
.

**Comment by ZeroDot1** [comment]
`[user@work ~]$ pyfunceble -w 32 --plain -a --syntax -f https://raw.githubusercontent.com/PyFunceble/ci_test/master/test.list
`

**Comment by spirillen** [comment]
> `[user@work ~]$ pyfunceble -w 32 --plain -a --syntax -f https://raw.githubusercontent.com/PyFunceble/ci_test/master/test.list `

:wink:   Yes that is a little fast test file we build, and it is actually exactly big enough for testing the `--ci-max-minutes 1`

**Comment by spirillen** [comment]
you are missing the log folder...

**Comment by ZeroDot1** [comment]
@spirillen Hm, I put all the files and folders in the zip file that was created, there is no extra folder with log files.

**Comment by spirillen** [comment]
:VERY VERY BIG EYES: and if you are using the "default" 
```shell
find $PYFUNCEBLE_OUTPUT_LOCATION/ -type f -iname debig.log
```

**Comment by spirillen** [comment]
and you are sure you are using pyf 4.x?? as the __pyfunceble_loggging__ have been default sine 4.0.0a1

**Comment by ZeroDot1** [comment]
Yes I am using the latest version and I tried the loglevel flag in the configuration. The log files were not created. I can't find any folder with the log files in the whole file system.

```py
[user@work ~]$ pyfunceble --version
/home/user/.local/lib/python3.9/site-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.25.11) or chardet (4.0.0) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
pyfunceble 4.0.2b5.dev (Blue Duckling: Chestnut)
[user@work ~]$ 

```

**Comment by spirillen** [comment]
Just curious, does this also happens with the stable 4.0.1?

**Comment by ZeroDot1** [comment]
@spirillen @funilrys  If I pass the flag additionally as argument the log files are created. I left it extra enabled in the configuration, but obviously this has no effect on the functionality.
The files are only created when I pass the argument manually. 

I quickly packed all the files into another zip file. [output.zip](https://github.com/funilrys/PyFunceble/files/7577142/output.zip)

`[user@work ~]$ pyfunceble -w 32 --plain -a --syntax --logging-level debug -f https://raw.githubusercontent.com/PyFunceble/ci_test/master/test.list`

**Comment by ZeroDot1** [comment]
> Just curious, does this also happens with the stable 4.0.1?

I have not tried the stable version, I have 90% only testing/alpha/ and beta software running on my system.

**Comment by ZeroDot1** [comment]
I think something is not working as it should. Maybe the program got infected with Corona :D

**Comment by spirillen** [comment]
Well it is made in the south but the south west not south east (AUS) :smirk:

**Comment by ZeroDot1** [comment]
OK, I just tested the non DEV version.
With the non DEV_Version no files are created.
I think it can't be my software/system, I have installed all available updates in my Arch Linux normally.
[output_new_configuration.zip](https://github.com/funilrys/PyFunceble/files/7586574/output_new_configuration.zip)
[output_old_configuration.zip](https://github.com/funilrys/PyFunceble/files/7586575/output_old_configuration.zip)

**Comment by ZeroDot1** [comment]
I noticed while checking the data that data is generated but it is not sorted into the folders.
![Bildschirmfoto vom 2021-11-23 09-26-20](https://user-images.githubusercontent.com/28985171/142991408-a8fb7898-e862-4069-8210-7c9d254a781b.png)

**Comment by ZeroDot1** [comment]
Sorry that I always find such special little problems.  :heart:

**Comment by spirillen** [comment]
> Sorry that I always find such special little problems. heart

```bash
sed 's/Sorry that I always find such special little problems/I love that I always find these special challanges'
```

**Comment by funilrys** [comment]
Okay, I finally found it .... 

If you have the following in your configuration:

```yaml
cli_testing
  file_generation:
    plain: true
    hosts: false
```

Then, when you use `--plain`, you are actually disabling it ... Meaning that pyfunceble loads:

```yaml
cli_testing
  file_generation:
    plain: false
    hosts: false
```

Meaning that no file is actually generated. The solution is just to remove `--plain` because the default value is `cli_testing.file_generation.plain: true` 😄 

Default: https://github.com/funilrys/PyFunceble/blob/dc72bd82ed4c5017d20a5962b838200bec50195c/PyFunceble/data/infrastructure/.PyFunceble_production.yaml#L184

@spirillen, I'm sure we discussed this in the past ...

**Comment by spirillen** [comment]
> @spirillen, I'm sure we discussed this in the past ..

I honestly did not notice this... But you are right long a long, setting a ` --feture` in the test arguments will counter the value set in the default pyfunceble configurations.

But I'm not the only one who overlooked it to begin with :wink:  https://github.com/funilrys/PyFunceble/issues/274#issuecomment-974225313 + https://github.com/funilrys/PyFunceble/issues/274#issuecomment-974603553 :wink: *whistle*

**Comment by ZeroDot1** [comment]
Hi @funilrys @spirillen, 

Oh man, sometimes I am sooo stupid, I should have noticed this error.

I think it would be good to add a function that informs users about misconfigurations.

Sorry that I have produced so much content here to this nonsense.

**Comment by funilrys** [comment]
@ZeroDot1, since you mentioned this, the commit that closed this issue introduced a way for me to provide some message to the end-user.

From now on, PyFunceble will produce the following when this "error" is seen:

![image](https://user-images.githubusercontent.com/14127285/143680292-696baf29-5623-42ce-957e-ba086dbd8f71.png)

If you have any other suggestions, feel free to share them in #277 !

Stay safe and healthy!

**Comment by ZeroDot1** [comment]
Hi @funilrys this is perfect, thank you very much.
